sway-lib-std/src/auth.sw

```diff
@@ -177,3 +177,34 @@ pub fn caller_address() -> Result<Address, AuthError> {
         None => Err(AuthError::CallerIsInternal),
     }
 }
+
+/// Get the current predicate's id when called in an internal context.
+/// 
+/// # Returns
+///
+/// * [Address] - The address of this predicate.
+///
+/// # Reverts
+///
+/// * When called not in a predicate context.
```

Comment:
```suggestion
/// * When called outside of a predicate program.
```

I think this is easier to understand than having the user know what a context is or if they're in the correct one.